src/js/app.js

```diff
@@ -0,0 +1,69 @@
+let myLibrary = [];
+
+function Book(title, author, pages, read) {
+  this.title = title;
+  this.author = author;
+  this.pages = pages;
+  this.read = read;
+}
+
+const addBookToLibrary = () => {
+  const title = document.getElementById('title').value;
+  const author = document.getElementById('author').value;
+  const pages = document.getElementById('pages').value;
+
+  const read = document.getElementById('read').checked;
+
+  const newBook = new Book(title, author, pages, read);
+  myLibrary.push(newBook);
+  localStorage.setItem('data-store', JSON.stringify(myLibrary));
+  window.location.reload();
+};
+
+const tableBody = document.querySelector('.table-body');
+
+const renderBook = () => {
+  myLibrary = JSON.parse(localStorage.getItem('data-store'));
+  if (myLibrary.length > 0) {
+    for (let i = 0; i < myLibrary.length; i += 1) {
```

Comment:
Kindly check out this part of the code. It raises a `type error` on the property `length`